install/installer/pkg/components/public-api-server/service.go

```diff
@@ -10,10 +10,6 @@ import (
 
 func service(ctx *common.RenderContext) ([]runtime.Object, error) {
 	return common.GenerateService(Component, map[string]common.ServicePort{
-		DebugPortName: {
```

Comment:
Thanks for removing this. It was the next for me to clean up after I landed the "debug/built-in server" last Friday.